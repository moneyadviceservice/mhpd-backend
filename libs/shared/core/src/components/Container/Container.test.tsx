import React from 'react';
import { render, screen } from '@testing-library/react';
import { Container } from '.';

describe('Container component', () => {
  it('renders correctly', () => {
    render(<Container data-testid="container">Container</Container>);
    const container = screen.getByTestId('container');
    expect(container).toMatchSnapshot();
  });
});
