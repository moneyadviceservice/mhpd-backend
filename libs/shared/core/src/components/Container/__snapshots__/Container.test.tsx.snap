// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container component renders correctly 1`] = `
<div
  class="container-auto"
  data-testid="container"
>
  Container
</div>
`;
